gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/DesktopApplicationTest.kt

```diff
@@ -366,6 +367,77 @@ class DesktopApplicationTest : GradlePluginTestBase() {
         }
     }
 
+    @Test
+    fun testMacLayeredIcon() {
+        Assumptions.assumeTrue(currentOS == OS.MacOS)
```

Comment:
I suppose that these tests will only run with SDK 26, please add an assumption for that as well.